src/statistics_tracking/WorldStatsTracker.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+/// <summary>
+///   Relays statistics about the world and the player to the organelle unlocks system (and later achievements)
+/// </summary>
+[UseThriveSerializer]
+public class WorldStatsTracker
+{
+    [JsonProperty]
+    public SimpleStatistic TotalEngulfedByPlayer { get; private set; } = new(StatsTrackerEvent.PlayerEngulfedOther);
+
+    [JsonProperty]
+    public SimpleStatistic TotalPlayerDeaths { get; private set; } = new(StatsTrackerEvent.PlayerDied);
+
+    [JsonProperty]
+    public ReproductionStatistic PlayerReproductionStatistic { get; private set; } = new();
+
+    public IStatistic[] CollectStatistics()
```

Comment:
To avoid extra list allocations, could this have an internal Lazy variable that holds this list so that it is only constructed once per tracker instance?